package.json

```diff
@@ -17,10 +17,11 @@
     "end-to-end",
     "e2e"
   ],
-  "author": "Kent C. Dodds <me@kentcdodds.com> (https://kentcdodds.com)",
+  "author": "Kent C. Dodds <kentcdodds@gmail.com> (https://kentcdodds.com)",
```

Comment:
Let's keep this as me@kentcdodds.com

Reply:
I just copied it over from `generator-kcd-oss`, will change it again 🙂